packages/snap-preact-components/src/components/Organisms/Autocomplete/Autocomplete.tsx

```diff
@@ -74,6 +74,7 @@ const CSS = {
 				flexDirection: 'column',
 				flex: `1 1 auto`,
 				maxWidth: `${vertical || horizontalTerms ? 'auto' : '150px'}`,
+				minWidth: '150px',
```

Comment:
I think this should be fine... But can you think of any negatives from introducing this min-width? I'm pretty sure we are preventing the render of `.ss__autocomplete__terms` when there are no terms right?

Reply:
I _think_ it should be fine. I did a quick search in implementations repo for .ss__autocomplete__terms to see how they are using it, and didn't see any conflicting or alarming use cases. I pulled down a site and tested it and all looked good.